libr/util/udiff.c

```diff
@@ -38,6 +38,129 @@ R_API int r_diff_set_delta(RDiff *d, int delta) {
 	return 1;
 }
 
+typedef struct levrow {
+	ut32 *changes;
+	ut32 start, end;
+} Levrow;
+
+static void lev_matrix_free(Levrow *matrix, ut32 len) {
+	ut32 i;
+	for (i = 0; i < len; i++) {
+		if (matrix[i].changes) {
```

Comment:
this changes pointer is never touched, freeing NULL is legit, so i would simplify that by removing the conditional blocks around